{"uuid":"dfe3d024-8ba5-4f58-bea1-23c0775c2e32","historyId":"ce926dd725fa848292bc8fb6bc0caa7d","fullName":"Array: The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","labels":[{"name":"tag","value":"arrays"},{"name":"tag","value":"scenario_016"},{"name":"host","value":"DESKTOP-RGL4EEL"},{"name":"thread","value":"12832@DESKTOP-RGL4EEL.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/janga/eclipse-workspace/DSAlgosample/src/test/resources/Features/UC05_Arrays.feature"},{"name":"package","value":"src.test.resources.Features.UC05_Arrays_feature.Array"}],"links":[],"name":"The user is presented with error message for code with invalid syntax in tryEditor for Arrays in Python page","status":"passed","stage":"finished","steps":[{"name":"Given  The user is in a page having an tryEditor with a Run button to test","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1680050516491,"stop":1680050516650},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1680050516651,"stop":1680050516931},{"name":"And  The user clicks on run button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1680050516932,"stop":1680050517084},{"name":"Then  The user should be presented with error message as \"NameError: name 'hello' is not defined on line 1\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1680050517085,"stop":1680050517099}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1680050516491,"stop":1680050517099}